www/gatsby-config.js

```diff
@@ -282,6 +284,16 @@ module.exports = {
         nodeTypes: [`StartersYaml`],
       },
     },
+    {
+      resolve: `gatsby-plugin-sentry`,
+      options: {
+        dsn: `https://2904ad31b1744c688ae19b627f51a5de@sentry.io/1471074`,
+        release: git.long(),
+        environment: process.env.NODE_ENV,
+        enabled: (() =>
+          [`production`, `stage`].indexOf(process.env.NODE_ENV) !== -1)(),
```

Comment:
is IIFE needed here?

Reply:
I didn't test it but the idea I believe is to only send a true or false to the browser not a function (which wouldnt work anyways).

Reply:
wouldn't
```js
enabled: [`production`, `stage`].indexOf(process.env.NODE_ENV) !== -1)
```
do the same? ;)

Reply:
Yeah, I just copied the README sample code

Reply:
We should probably change the README 😅